src/vtt-segment-loader.js

```diff
@@ -279,6 +279,16 @@ export default class VTTSegmentLoader extends SegmentLoader {
 
     this.mediaSecondsLoaded += segment.duration;
 
+  /**
+   * The below code helps in removing double captions
+   */
+    if (segmentInfo.cues.length) {
+      let lastCueEndTime = segmentInfo.cues[segmentInfo.cues.length - 1].endTime;
+      let firstCueEndTime = segmentInfo.cues[0].endTime;
+
+      this.remove(firstCueEndTime, lastCueEndTime);
```

Comment:
The values do change for segmentInfo cues,  So I used let..But I did change the 3 lines code into 1 line and updated the comment above the code changes as you suggested